src/modify.js

```diff
@@ -1,6 +1,9 @@
 import get from 'lodash/get';
 import mergeWith from 'lodash/mergeWith';
 
+const WARNING_TYPES = {
+  CHANGE_FIELD_NOT_FOUND: 'CHANGE_FIELD_NOT_FOUND',
```

Comment:
```suggestion
  CHANGED_FIELD_NOT_FOUND: 'CHANGED_FIELD_NOT_FOUND',
```

Reply:
🤔  CHANGED assumes it was indeed changed. CHANGE is the action (change field), which actually did not happen because it was not found.

Reply:
I see, maybe `FIELD_TO_CHANGE_NOT_FOUND`? 😅

Reply:
no need to change, I just find "change field" (action) and "not found" a bit weird. It's the "field" that wasn't found, not the "change field" action

Reply:
there you go :D